### ISSUE TITLE
Transaction history for users, buckets and groups

### ISSUE THREAD

**Issue by jessykate** [cobudget/cobudget#108]
![enspiral](https://cloud.githubusercontent.com/assets/59882/24154624/00896356-0e52-11e7-97c0-bc3b8b375ebe.jpg)

@jlopker my initial thinking was to just get a table of the allocation history for the group loading, and then to try and make a chart something like the attached image.

i thought the funders tab might be a good place for it because it's where the money is added/allocated in the first place, but that was just a thought. it could go on the stats tab too...

**Comment by jessykate** [comment]
![cobudget_](https://cloud.githubusercontent.com/assets/59882/24362042/d51d1532-130b-11e7-8882-fffb12c8d9f1.jpg)

@jlopker this is looking amazing. Love the new table and the `type` column. Ideas/suggestions:

- [ ] add balance column?
- [ ] enable pagination (some groups have 2 years of history at this point)
- [ ] remove star icon in group totals header (also on stats tab)
- [ ] chart title - what should it be? ("by date" is implicit in the temporal x-axis)
- [ ] does "transaction list" need a heading? isn't it kind of implied?
- [ ] y-axis label should be "balance" not "allocations" (and then we don't need the legend i guess?)
- [ ] can we get rid of the highcharts.com link?

**Comment by jessykate** [comment]
yes! just a couple small things around decimals and rounding:

it looks like the balance column is using the `floor` of the number (ie, rounding down), whereas the amount column is preserving the decimal values, which looks funny:

![cobudget_](https://cloud.githubusercontent.com/assets/59882/24461097/70901664-14a0-11e7-82a6-79db305e5664.jpg)

but it also creates a discrepancy between the "balance in group" total, and the final balance value of the table:

![cobudget_](https://cloud.githubusercontent.com/assets/59882/24461166/9b8fb478-14a0-11e7-9816-1859ddf3ae56.jpg)


basically i think we should just preserve decimals in all these places (including the "balance in group" value) so we have a precise number, which is what admins will need anyway.  

otherwise i am good to go on this branch!!
